src/Analyzers/CSharp/Tests/FixReturnType/FixReturnTypeTests.cs

```diff
@@ -2,353 +2,442 @@
+                class C
+                {
+                    int M() => 1;
+                }
+                """);
         }
 
         [Fact, WorkItem(47089, "https://github.com/dotnet/roslyn/issues/47089")]
         public async Task ExpressionAndReturnTypeAreVoid()
         {
-            await TestMissingInRegularAndScriptAsync(
-@"class C
-{
-    void M()
-    {
-        return Console.WriteLine()[||];
-    }
-}");
+            var markup = """
+                using System;
+
+                class C
+                {
+                    void M()
+                    {
+                        {|CS0127:return|} Console.WriteLine();
+                    }
+                }
+                """;
+            await VerifyCS.VerifyCodeFixAsync(markup, markup);
         }
 
         [Fact, WorkItem(53574, "https://github.com/dotnet/roslyn/issues/53574")]
         public async Task TestAnonymousTypeTopLevel()
         {
-            await TestInRegularAndScript1Async(
-@"class C
-{
-    public void Method()
-    {
-        [|return|] new { A = 0, B = 1 };
-    }
-}",
-@"class C
-{
-    public object Method()
-    {
-        return new { A = 0, B = 1 };
-    }
-}");
+            await VerifyCS.VerifyCodeFixAsync("""
+                class C
+                {
+                    public void Method()
+                    {
+                        {|CS0127:return|} new { A = 0, B = 1 };
+                    }
+                }
+                """, """
+                class C
+                {
+                    public object Method()
+                    {
+                        return new { A = 0, B = 1 };
+                    }
+                }
+                """);
         }
 
         [Fact, WorkItem(53574, "https://github.com/dotnet/roslyn/issues/53574")]
         public async Task TestAnonymousTypeTopNested()
         {
-            await TestInRegularAndScript1Async(
-@"class C
-{
-    public void Method()
-    {
-        [|return|] new[] { new { A = 0, B = 1 } };
-    }
-}",
-@"class C
-{
-    public object Method()
-    {
-        return new[] { new { A = 0, B = 1 } };
-    }
-}");
+            await VerifyCS.VerifyCodeFixAsync("""
+                class C
+                {
+                    public void Method()
+                    {
+                        {|CS0127:return|} new[] { new { A = 0, B = 1 } };
+                    }
+                }
+                """, """
+                class C
+                {
+                    public object Method()
+                    {
+                        return new[] { new { A = 0, B = 1 } };
+                    }
+                }
+                """);
+        }
+
+        [Fact, WorkItem(64901, "https://github.com/dotnet/roslyn/issues/64901")]
+        public async Task ReturnString_ValueTask()
+        {
+            await new VerifyCS.Test
+            {
+                TestCode = """
+                using System.Threading.Tasks;
```

Comment:
nit: my personal preference would be to have these indented one level further, to distinguish the properties from the contents.  But it's NBD to do it this way.